src/main/java/calculator/domain/calculator/Number.java

```diff
@@ -0,0 +1,44 @@
+package calculator.domain.calculator;
+
+import java.util.Objects;
+
+public class Number {
+
+    private final int number;
+
+    public Number(int number) {
+        validateNegative(number);
+        this.number = number;
+    }
+
+    public Number(String input) {
+        this(Integer.parseInt(input));
+    }
+
+    private void validateNegative(int number) {
+        if (number < 0) {
+            throw new NegativeInputException();
+        }
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public static Number add(Number number1, Number number2) {
+        return new Number(number1.getNumber() + number2.getNumber());
```

Comment:
동일한 클래스에선 getter를 사용하기 보단 직접 접근하는것이 어떨까요 >? 
```suggestion
        return new Number(number1.number + number2.number);
```